airbyte-webapp/src/components/connection/ConnectionInfoCard/ConnectionInfoCard.test.tsx

```diff
@@ -32,9 +32,9 @@ const TestWrapperWithAutoDetectSchema: React.FC<React.PropsWithChildren<Record<s
 );
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const { StatusMainInfo } = require("./StatusMainInfo");
+const { ConnectionInfoCard } = require("./ConnectionInfoCard");
 
-describe("<StatusMainInfo />", () => {
+describe("<connectionInfo />", () => {
```

Comment:
One thing I like to do for test names is `describe(`${ConnectionInfoCard.name}`, () => {`

So that renaming the variable will automatically update the name of the test.